qrisk/stats.py

```diff
@@ -15,13 +15,10 @@
 
 from __future__ import division
 
-from collections import OrderedDict
-from functools import partial
-
 import pandas as pd
 import numpy as np
 import scipy as sp
-import scipy.stats
+import scipy.stats  # noqa
```

Comment:
the `{}` was like a format string, or `%s`, it meant "put the name of the function(s) you want to use here